test/clj_kondo/discouraged_tag_test.clj

```diff
@@ -0,0 +1,46 @@
+(ns clj-kondo.discouraged-tag-test
```

Comment:
Thanks. I think using assert-submaps2 gives better error message, so let's use that. I'm slowly phasing assert-submaps(1) out.

discouraged-var also supports setting the :level per var. I think that would be good to support here as well. 
Happy to help or take over, when you're stuck.

Reply:
Added in  e9538ec :)